src/units.cpp

```diff
@@ -506,9 +532,18 @@ bool Units::equivalent(const UnitsPtr &units1, const UnitsPtr &units2)
     if ((units1->isImport()) || (units2->isImport())) {
         return false;
     }
-
-    UnitsMap units1Map = createUnitsMap(units1);
-    UnitsMap units2Map = createUnitsMap(units2);
+    if ((units1->requiresImports()) || (units2->requiresImports())) {
+        return false;
+    }
+    bool isValid = true;
```

Comment:
No need to initialise `isValid`.

Reply:
... hmm, unless the compiler is not clever enough to realise that `createUnitsMap()` initialises it...

Reply:
Done.